airflow/contrib/hooks/datadog_hook.py

```diff
@@ -47,26 +47,15 @@ def __init__(self, datadog_conn_id='datadog_default'):
         # for all metric submissions.
         self.host = conn.host
 
-        if self.api_key is None:
-            raise AirflowException("api_key must be specified in the "
-                                   "Datadog connection details")
-        if self.app_key is None:
-            raise AirflowException("app_key must be specified in the "
-                                   "Datadog connection details")
-
         self.log.info("Setting up api keys for Datadog")
-        options = {
-            'api_key': self.api_key,
-            'app_key': self.app_key
-        }
-        initialize(**options)
+        initialize(api_key=self.api_key, app_key=self.app_key)
 
     def validate_response(self, response):
         if response['status'] != 'ok':
             self.log.error("Datadog returned: %s", response)
             raise AirflowException("Error status received from Datadog")
 
-    def send_metric(self, metric_name, datapoint, tags=None):
+    def send_metric(self, metric_name, datapoint, tags=None, type_=None, interval=None):
```

Comment:
typo .. `type_` -> `type`

Reply:
or may be change the name to `metric_type`

Reply:
The option is called `type` in the datadog docs, and I appended an underscore to avoid shadowing the `type` builtin in python.